src/Constraint/MultiConstraint.php

```diff
@@ -126,12 +129,16 @@ public function getPrettyString()
      */
     public function __toString()
     {
+        if ($this->string) {
```

Comment:
Should compare against null.

Reply:
I used the same syntax than `prettyString` for consistency. Tell me if you want something cleaner, I'll open a PR